packages/node/src/lib/construct-payload.ts

```diff
@@ -54,6 +54,11 @@ export interface LogOptions {
    */
   fireAndForget?: boolean;
 
+  /**
+   * If true, the errors and other logs will be displayed in console.
+   */
+  logger?: boolean;
```

Comment:
What if we allow this option to consume a `Logger` class (with typescript enforcement from our lib), as an option.

Something like `logger?: Logger | boolean;`. 

We can then fallback to our own logger implementation if it's not passed, but the SDK consumer passes `true`.